### DISCUSSION TITLE
TIC-80 on Emulationstation

### DISCUSSION THREAD

**Discussion by revdeluxe** [nesbox/TIC-80#1728] [Ideas]
**So you've read it RIGHT... :)**
- Please use the #1731 thread

**First off the cons**

-  Doesn't recognize controller at first try but i works later on(im using PS3 ds3 bluetooth controller on my linux mint)
-  You need Emulationstation/Retropie for this to work or not. just `$ tic80` or launch the app via exec.(that definately defeats the purpose of this thread...)
-  For some reason --fullscreen launch options renders a black screen. so don't use it i guess if it works fine on your system then why not...
-  Then some minor DB editing. welp expect this one.

then the pros
-   puts every game in your system in one place 'Emulationstation/Retropie/other launchers'
-   the controller works(in my case)
-   cosmetics/themes/aesthetic... don't forget that ESTHEMES exist. if you want a launcher that fits in your setup. of course other than using `surf`

**Firstly (varies on OS)**

-   find the es_system.cfg
-   `folder` in the tic80 command line interface *COPY THE PATH YOU'LL NEED IT*
-   COPY then PASTE the code inside the <systemlist></systemlist> tag in es_system.cfg
-  Inside the folder make a folder named GAMES. of course put all your .tic cartridges in that folder
-  (OPTIONAL) go an extra mile for beautification in the menu, as you can see in my screenshot the TIC-80 is in plain white...
```
    <system>
        <name>TIC-80</name>
        <fullname>Virtual Console TIC-80</fullname>
        <path>##--------PASTE THE PATH YOU'D COPIED EARLIER---------##</path>
        <extension>.tic</extension>
        <command>tic80 --skip --cmd="cd games & load %BASENAME% & run"</command>
        <platform>TIC-80</platform>
        <theme>tic80</theme>
    </system>
```

**CLOSING REMARKS:**
> this is a great way to play since you can access the games with ease...
> now this made me think. please add multiplayer
**the controller disconnected that's why the 4 block line in tetris was wildly spinning**
**Here are some clips/screenshot**
![TIC80](https://user-images.githubusercontent.com/38414584/145441884-1be73d00-fa5c-4166-a4be-423d3e24de06.png)

https://user-images.githubusercontent.com/38414584/145441984-c1241020-67c7-4b10-907e-c9621f78614f.mp4

**Comment by joshgoebel** [comment]
> So you've read it RIGHT... :)

If you pretend not all of us know what emulationstation is then a nice sentence or two introduction would be a lot nicer I think and make this issue accessible for more readers. :-)  I'm quite involved with TIC-80 but only very, very roughly know anything about Emulation station (that it's some emulation platform).

**Reply by revdeluxe** [reply]
Oh, thanks... totally forgot that... hahahahaha.
Right... i'll make this long in a good way.
